app/src/main/res/drawable/ic_audio_lang_24px.xml

```diff
@@ -1,6 +1,7 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
   android:width="24dp"
   android:height="24dp"
+  android:autoMirrored="true"
```

Comment:
@BenHenning Please confirm.

Reply:
Per https://material.io/design/usability/bidirectionality.html#mirroring-elements under 'Passage of Time' is this PR actually correct @rt4914? We shouldn't be mirroring progress bars or player buttons if they represent time since time is conventionally not mirrored in RTL contexts (unless I'm misunderstanding the bidi advice from that page).

Reply:
@BenHenning We have not mirrored player buttons. But this icon is `language-change-icon` which I think might need some changes basically the background needs to be flipped. Also we have mirrored only 5 icons mentioned here: https://github.com/oppia/oppia-android/pull/3593/files

Reply:
@MohamedMedhat1998 Can you help us out in deciding what should be RTL behaviour of this icon:

<img width="200" alt="Screenshot 2021-08-04 at 10 38 37 PM" src="https://user-images.githubusercontent.com/9396084/128224235-836d9d97-f649-436d-b57d-7e72a2841d6c.png">

1. If we do not mirror it it will look exactly same in RTL too.
2. If we mirror it fully than the text will be mirrored too which means that the character in foreign language will also get mirrored meaning it won't be a valid character anymore. So definitely we cannot do this.
<img width="286" alt="Screenshot 2021-08-04 at 10 41 16 PM" src="https://user-images.githubusercontent.com/9396084/128224528-2a3aaa8c-85ad-4534-9bd8-c9b851819558.png">
3. Another option we have is to mirror the bubble/background only and keep the text exactly same for RTL.


What are your thoughts? What would be the best solution here?

Reply:
Thanks @rt4914 for pointing out that I was mistaking this icon. Clearing my review for now since my codeowner approval isn't needed, and per our discussion this morning.

Reply:
> @MohamedMedhat1998 Can you help us out in deciding what should be RTL behaviour of this icon:
> 
> <img alt="Screenshot 2021-08-04 at 10 38 37 PM" width="200" src="https://user-images.githubusercontent.com/9396084/128224235-836d9d97-f649-436d-b57d-7e72a2841d6c.png">
> 
> 1. If we do not mirror it it will look exactly same in RTL too.
> 2. If we mirror it fully than the text will be mirrored too which means that the character in foreign language will also get mirrored meaning it won't be a valid character anymore. So definitely we cannot do this.
> 
> <img alt="Screenshot 2021-08-04 at 10 41 16 PM" width="286" src="https://user-images.githubusercontent.com/9396084/128224528-2a3aaa8c-85ad-4534-9bd8-c9b851819558.png">
> 
> 3. Another option we have is to mirror the bubble/background only and keep the text exactly same for RTL.
> What are your thoughts? What would be the best solution here?

Hi everyone. Thanks for mentioning me :)    
I think I haven't seen any mirrored icons throughout my life, all the icons remain the same but only their position changes.  
This doesn't apply to the navigation icons because they are always mirrored.  
.
I have checked the YouTube app and all the icons are the same in both locales. I have also checked Chome app and found only one mirrored icon, which is the "share" icon, but it looked weird to me btw. Also, the cellular signal strength icon found in the action bar of the device doesn't change.   
So to summarize my opinion  
1. Generally, don't mirror the icons.
2. Mirror only the navigation icons.  


According to this case specifically, if the player remains the same (which is the correct option according to what @BenHenning  has shared and to my experience with the players) then leave the icon as is. (don't mirror the text nor the background)

Reply:
@veena14cs As per @MohamedMedhat1998 's comment can you undo this change.